releasing/docker-run.sh

```diff
@@ -72,8 +72,7 @@ case "$TARGET_OS" in
   ;;
   centos)
```

Comment:
Maybe it's worth adding a comment here explaining that CentOS 8 is EOL but CentOS 7 is "alive" until 2024. That will avoid us (others) wondering in the future why is an "older" version tested but not the "newer".

Reply:
I updated the releasing/README with that information